pkg/ingester/transfer.go

```diff
@@ -162,3 +163,102 @@ func fromWireChunks(wireChunks []client.Chunk) ([]*desc, error) {
 	}
 	return descs, nil
 }
+
+// Transfer finds an ingester in PENDING state and transfers our chunks to it.
+// Called as part of the ingester shutdown process.
+func (i *Ingester) Transfer() error {
```

Comment:
`Transfer` is outward; `TransferChunks` is inward.  Could we pick names that make this clearer?